digitalocean/digitaloecan-droplets.tf

```diff
@@ -0,0 +1,57 @@
+terraform {
+  required_version = ">= 0.13"
+  required_providers {
+    digitalocean = {
+      source  = "digitalocean/digitalocean"
+      version = "2.5.1"
+    }
+    ct = {
+      source  = "poseidon/ct"
+      version = "0.7.1"
+    }
+    template = {
+      source  = "hashicorp/template"
+      version = "~> 2.2.0"
+    }
+    null = {
+      source  = "hashicorp/null"
+      version = "~> 3.0.0"
+    }
+  }
+}
+
+resource "digitalocean_ssh_key" "first" {
+  name       = var.cluster_name
+  public_key = var.ssh_keys.0
+}
+
+resource "digitalocean_custom_image" "flatcar" {
+  name   = "flatcar-stable-2605.12.0"
```

Comment:
Same comment as in the docs, the version number should be a variable.

Reply:
Done